pkgs/tools/networking/xdp-tools/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, libbpf, elfutils, libelf, zlib, libpcap
+, clang, llvm, gnumake, gcc, pkgconfig, m4, emacs-nox, wireshark-cli
+}:
+
+let
```

Comment:
Normally, `let in` is not used. You can write directly in the `stdenv.mkDerivation`.So `projectSha256` is not necessary either.

Reply:
I added `projectSha256` and `version` to expose these values, easily for the future changes. I understand that `version` variable is used correctly, but I should move `src` directly to the `stdenv.mkDerivation`?